<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="D1head_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="D1booker_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>200</width>
     <height>225</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="book_title">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
  </widget>
  <widget class="QLabel" name="book_author">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>author</string>
   </property>
  </widget>
  <widget class="QLabel" name="book_year">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QLabel" name="book_genre">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="book_description">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>650</width>
     <height>230</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>description</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="book_quantity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>quantity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="order_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Заказать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
